### DISCUSSION TITLE
v0.10-dev

### DISCUSSION THREAD

**Discussion by pablohashescobar** [makeplane/plane#1787] [Announcements]
# Release v0.10-dev

### ✈️ **What’s new?**

We're thrilled to unleash our quick succession major release, loaded with intuitive updates that will take Plane to the next level of awesomeness! 💫 

****Cross-Project issue property issue linking****

With this new feature, you can now effortlessly link any issue from any project within your workspace to the current project. Whether it's a sub-issue, blocked issue, or blocked by issue, managing issues and collaborating across projects is now easier with a single toggle.

![image](https://github.com/makeplane/plane/assets/118773738/5f12641c-ec9b-4828-a595-731faf94e606)

**([#1609](https://github.com/makeplane/plane/pull/1609), [#1612](https://github.com/makeplane/plane/pull/1612), [#1625](https://github.com/makeplane/plane/pull/1625))**

**Say Hello to the Revamped My Issues Page! 🚀 🔍**

We are excited to present the all-new **My Issues** page with enhanced customization options, empowering you to take full control of your issue management! With this revamp, we've introduced a host of powerful features.

![image](https://github.com/makeplane/plane/assets/118773738/ee2a7bc1-af1c-41f6-9df4-aa159de69511)

- **👁️ Customize View Options:** Enjoy the freedom to personalize your issue view exactly the way you want.
- **🔍 Add Filters for Efficient Issue Sorting:** Now, you can effortlessly add filters to your My Issues, allowing you to quickly find and focus on the tasks that matter most to you.
- **🛠️ Choose Your Preferred Layout:** Now, you can effortlessly select your preferred issue view layout, whether it's a comprehensive and detailed layout or a simplified and streamlined one.
- 👀 **Stay Updated on Your Set of Issues:** Stay on top of your tasks by selecting your view based on issues assigned to you or those created by you.

([#1666](https://github.com/makeplane/plane/pull/1666), [#1681](https://github.com/makeplane/plane/pull/1681), [#1726](https://github.com/makeplane/plane/pull/1726), [#1727](https://github.com/makeplane/plane/pull/1727), [#1733](https://github.com/makeplane/plane/pull/1733), [#1753](https://github.com/makeplane/plane/pull/1753), [#1754](https://github.com/makeplane/plane/pull/1754))

**🫂 Member’s Profile Analytics**

Now, you have access to a comprehensive view of members' activities, covering issue creation, assignments, and subscriptions, all conveniently located in one place.

 🙌 With customizable layouts, including List and Kanban views, plus powerful groupings, ordering, and display options, you can now gain deeper insights into each member's contributions. 

📊 Our robust filtering options provide a transparent, organized workflow, enhancing collaboration and team efficiency!   

([#1682](https://github.com/makeplane/plane/pull/1682), [#1698](https://github.com/makeplane/plane/pull/1698), [#1710](https://github.com/makeplane/plane/pull/1710), [#1728](https://github.com/makeplane/plane/pull/1728), [#1733](https://github.com/makeplane/plane/pull/1733), [#1737](https://github.com/makeplane/plane/pull/1737), [#1745](https://github.com/makeplane/plane/pull/1745), [#1747](https://github.com/makeplane/plane/pull/1747), [#1749](https://github.com/makeplane/plane/pull/1749), [#1751](https://github.com/makeplane/plane/pull/1751), [#1752](https://github.com/makeplane/plane/pull/1752), [#1754](https://github.com/makeplane/plane/pull/1754), [#1765](https://github.com/makeplane/plane/issues/1765))

![image](https://github.com/makeplane/plane/assets/118773738/3236b783-2c56-40aa-98af-41f4e566c25d)

🧙‍♂️ ****Expanded Content Magic: Pages' Block View Personalization!****

By enabling the `Show full block content` option within the page, you'll experience pure magic as the expanded view is automatically applied to all blocks in all the pages within the project.

![image](https://github.com/makeplane/plane/assets/118773738/188ee338-a0b7-442b-87ac-61f925d3110a)

- **Enhanced Visibility:** Bid farewell to the hassle of clicking on each block individually. With this latest update, relish an expanded view of each block's content without the need for repetitive clicks.

Once you activate this feature, your preferred view will be consistently applied across all blocks in all pages within the project until you decide to disable it. ([#1652](https://github.com/makeplane/plane/pull/1652), [#1669](https://github.com/makeplane/plane/pull/1669), [#1673](https://github.com/makeplane/plane/pull/1673))

**✈️ Express Yourself with Emojis!**

Now, you can react like never before to issues and comments using fun emojis! 😄👍 Spread positivity, show support, and express yourself effortlessly with a click! Collaborating has never been this delightful! So, let your emoji thoughts flow and make it a breeze! ([#1674](https://github.com/makeplane/plane/pull/1674), [#1690](https://github.com/makeplane/plane/pull/1690), [#1692](https://github.com/makeplane/plane/pull/1692), [#1768](https://github.com/makeplane/plane/issues/1768))

![image](https://github.com/makeplane/plane/assets/118773738/f5913e21-dcfb-41d9-88d0-6dd993fca16f)

****🧹 Manage Sub-Issues with Ease!****

We've refined our Sub-Issues View to ensure smoother project management. Now, you can effortlessly toggle the view to show or hide sub-issues, allowing you to focus on what requires your attention. Experience a more organized project and enhance your productivity! 📝🧹     ([#1665](https://github.com/makeplane/plane/pull/1665), [#1679](https://github.com/makeplane/plane/pull/1679))

****🌈 Project Cover Image 📸: Let Your Projects Shine!****

Unleash your creativity with Project Cover Images! Add a touch of uniqueness to your projects by uploading captivating cover images. Make a lasting impression and express the essence of each project like never before. It's the perfect way to make your projects stand out in style! 🌟🖼️   ([#1668](https://github.com/makeplane/plane/pull/1668), [#1676](https://github.com/makeplane/plane/pull/1676), **[#1704](https://github.com/makeplane/plane/pull/1704)**)

**🔄 Efficient Project Reordering** 

Reorder projects effortlessly with drag & drop! Organize favorites, joined, and un-joined projects separately in the sidebar with our improved UX. Boost productivity and stay on top of your tasks. ([#1701](https://github.com/makeplane/plane/pull/1701), [#1725](https://github.com/makeplane/plane/pull/1725), [#1727](https://github.com/makeplane/plane/pull/1727), [#1736](https://github.com/makeplane/plane/pull/1736), [#1738](https://github.com/makeplane/plane/pull/1738), [#1744](https://github.com/makeplane/plane/pull/1744), [#1756](https://github.com/makeplane/plane/pull/1756))

 

### ⚙️ Improvements

- Added a tooltip for the notification icon in the collapsed sidebar. (**[#1605](https://github.com/makeplane/plane/pull/1605)**)
- Updated the border style of the automation settings. (**[#1605](https://github.com/makeplane/plane/pull/1605)**)
- Updated the cursor style for the free plane label in the sidebar. (**[#1605](https://github.com/makeplane/plane/pull/1605)**)
- Improved Issue Create Update by n+1, Issue Activity Get by n+1, and Cycles by n+1. (**[#1606](https://github.com/makeplane/plane/pull/1606), [#1689](https://github.com/makeplane/plane/pull/1689)**)
- Implemented proper error message display on the profile form submission. (**[#1611](https://github.com/makeplane/plane/pull/1611)**)
- System preference is now shown by default, reducing setup complexity. (**[#1616](https://github.com/makeplane/plane/pull/1616)**)
- User-selected themes are now saved and applied on subsequent visits. (**[#1616](https://github.com/makeplane/plane/pull/1616)**)
- Project leads are now automatically added as project members, even when they are not current project members. (**[#1627](https://github.com/makeplane/plane/pull/1627)**)
- Styled the created project modal, improving the visual appearance. (**[#1628](https://github.com/makeplane/plane/pull/1628)**)
- Revamped cycles list page design with empty states for improved usability. (**[#1633](https://github.com/makeplane/plane/pull/1633)**)
- The project details endpoint now returns total members, cycles, and modules, providing more comprehensive project information. (**[#1637](https://github.com/makeplane/plane/pull/1637)**)
- Updated the project identifiers length to accept 12 characters and added proper validations for character entries, ensuring data integrity. (**[#1636](https://github.com/makeplane/plane/pull/1636)**, **[#1638](https://github.com/makeplane/plane/pull/1638)**, **[#1643](https://github.com/makeplane/plane/pull/1643), [#1663](https://github.com/makeplane/plane/pull/1663), [#1755](https://github.com/makeplane/plane/pull/1755), [#1757](https://github.com/makeplane/plane/pull/1757)**)
- Updated the project issue link activity to return the URL on delete, improving the handling of issue links. (**[#1639](https://github.com/makeplane/plane/pull/1639)**)
- All the issue properties will now be enabled by default just after creating the project. (**[#1640](https://github.com/makeplane/plane/pull/1640)**)
- Added the ability for guests and viewers to comment, promoting collaboration and feedback from all project stakeholders. (**[#1650](https://github.com/makeplane/plane/pull/1650)**)
- Improved the global search endpoint for across-project search in a workspace, enabling users to quickly find information across multiple projects. (**[#1649](https://github.com/makeplane/plane/pull/1649)**)
- Enhanced UX for labels on issue cards with visible applied titles. ([#1653](https://github.com/makeplane/plane/pull/1653))
- Added Pagination to Notifications for better and organized visibility about all your updates. ([#1580](https://github.com/makeplane/plane/pull/1580), [#1678](https://github.com/makeplane/plane/pull/1678), [#1734](https://github.com/makeplane/plane/pull/1734))
- Improved search results positioning for a more intuitive experience. ([#1680](https://github.com/makeplane/plane/pull/1680))
- Added project identifier for each search result item. ([#1680](https://github.com/makeplane/plane/pull/1680))
- Enhanced the responsiveness of issue titles to match the screen responsiveness. ([#1683](https://github.com/makeplane/plane/pull/1683))
- Improved the UI for the Sibling Issues & Views widget's menu for a more intuitive user experience. ([#1699](https://github.com/makeplane/plane/pull/1699), [#1742](https://github.com/makeplane/plane/pull/1742))
- Added new colors to enhance the theming structures. ([#1727](https://github.com/makeplane/plane/pull/1727))
- Added bulk invites for members at the workspace level. ([#1530](https://github.com/makeplane/plane/pull/1530))
- Get notified on issues you create, regardless of role in the workspace. ([#1739](https://github.com/makeplane/plane/pull/1739), [#1740](https://github.com/makeplane/plane/pull/1740))
- Improved Create Project form with better styling. ([#1741](https://github.com/makeplane/plane/pull/1741))
- Leave workspaces effortlessly with the Leave option. ([#1746](https://github.com/makeplane/plane/pull/1746))
- Global CSS file now includes a primary color variable for seamless styling. ([#1748](https://github.com/makeplane/plane/pull/1748))
- Added non-existing states for project entities ([#1770](https://github.com/makeplane/plane/issues/1770))
- Improved alignment issue in comment activity ([#1771](https://github.com/makeplane/plane/issues/1771))
- Added consistent padding to match the sidebar's layout applied ([#1767](https://github.com/makeplane/plane/issues/1767))
- Implemented dynamic length title on hover, which allows it to expand and occupy available space while shrinking upon hovering to reveal the quick action icon in the sidebar. ([#1767](https://github.com/makeplane/plane/issues/1767))
- Show a message if dragging an un joined project in the sidebar. ([#1763](https://github.com/makeplane/plane/issues/1763))

### 🪲 Bug Fixes

- Fixed an issue where project selection was not working on the create issue modal. ([#1608](https://github.com/makeplane/plane/pull/1608))
- Resolved a bug related to custom search select query. ([#1610](https://github.com/makeplane/plane/pull/1610))
- Fixed the link form not submitting when pressing **`Enter`**. ([#1613](https://github.com/makeplane/plane/pull/1613))
- Fixed the issue activity for attachments, ensuring proper display of attachment-related activities. (**[#1626](https://github.com/makeplane/plane/pull/1626)**)
- Fixed the issue parent activity. ([#1629](https://github.com/makeplane/plane/pull/1629))
- Removed the blocker and blocked issue payload. ([#1629](https://github.com/makeplane/plane/pull/1629))
- Disabled older dates on the start date popup, preventing the selection of invalid dates. (**[#1631](https://github.com/makeplane/plane/pull/1631)**)
- Closed **`datepicker`** once the date is selected, improving the user interface of date selection. (**[#1631](https://github.com/makeplane/plane/pull/1631)**)
- Fixed the issue with the remove issue link activity not displaying the removed link, providing better clarity when issue links are removed. **([#1644](https://github.com/makeplane/plane/pull/1644))**
- Titles for issue links are now optional, allowing users the flexibility to include titles for issue links when desired. **([#1641](https://github.com/makeplane/plane/pull/1641))**
- Issue link validation is now fixed, ensuring accurate validation of issue links, preventing incorrect or broken links. **([#1645](https://github.com/makeplane/plane/pull/1645))**
- Fixed the issue with the create issue modal resetting when the project is changed, ensuring a consistent and seamless issue creation experience. (**[#1647](https://github.com/makeplane/plane/pull/1647))**
- Fixed the notification select related fields for n+1s, resolving issues related to notification selection. (**[#1646](https://github.com/makeplane/plane/pull/1646))**
- Fixed the display of API errors for AI responses. **(**[#1677](https://github.com/makeplane/plane/pull/1677)**)**
- Fetching states only when the dropdown is opened. ([#1684](https://github.com/makeplane/plane/pull/1684))
- Fixed the issue with filters for enhanced properties fetch. ([#1688](https://github.com/makeplane/plane/pull/1688))
- Fixed the issue with conditional checks for the create view form. ([#1691](https://github.com/makeplane/plane/pull/1691))
- The issue of workspace deletion when only one admin or member is present has been resolved.([#1700](https://github.com/makeplane/plane/pull/1700), [#1730](https://github.com/makeplane/plane/pull/1730))
- Added appropriate validations to the workspace name. ([#1702](https://github.com/makeplane/plane/pull/1702))
- Fixed the issue where Google authentication tab was not accepting width as a string. ([#1708](https://github.com/makeplane/plane/pull/1708))
- Fixed the issue with the image uploader by allowing it to accept PNG and JPEG file formats only. ([#1720](https://github.com/makeplane/plane/pull/1720))
- In the rich text editor, image uploads will now take 35% of the width instead of 100%. ([#1720](https://github.com/makeplane/plane/pull/1720))
- Fixed the mutation issue related to workspace member invitation. ([#1721](https://github.com/makeplane/plane/pull/1721))
- Moved the mutation of project invitations to the finally block. ([#1722](https://github.com/makeplane/plane/pull/1722))
- Issue with project invite has been fixed. ([#1760](https://github.com/makeplane/plane/issues/1760))
- The problem of project selection while creating an issue from the create issue form has been resolved. ([#1766](https://github.com/makeplane/plane/issues/1766))
- Prevented accidental toggling of notification read status. ([#1769](https://github.com/makeplane/plane/issues/1769))
- Resolved custom theme persistence after signing out. ([#1780](https://github.com/makeplane/plane/issues/1780))

<hr /><em>This discussion was created from the release <a href='https://github.com/makeplane/plane/releases/tag/v0.10-dev'>v0.10-dev</a>.</em>

**Comment by theetherGit** [comment]
@pablohashescobar I'm unable to understand why it's using my CPU & Memory to 100% on server while i update the project because of this i can't update it. I haven't faced this issue while upgrading to v0.8 & v0.9.

One more thing, can you please provide/suggest server size for this setup maybe my setup have less cpu or ram.

**Comment by pablohashescobar** [comment]
Hi @theetherGit, anything above 2GB should be good also can you stop all the containers and also restart the server maybe if you are still facing high memory and RAM usage.

**Reply by theetherGit** [reply]
Thank You, I'll check and update you if i face any problem.
